frontend/components/text2text/results/ContentEditableFeedbackTask.vue

```diff
@@ -81,6 +85,13 @@ export default {
       this.focus = status;
       this.$emit("on-change-focus", status);
     },
+    pastePlainText(event) {
+      if (this.focus && event.target.isContentEditable) {
+        event.preventDefault();
+        const text = event.clipboardData?.getData("text/plain");
```

Comment:
with the optional chaining `key?.key...` means that the const `text` could be equal to `undefined`, the user will be able to paste a value `undefined`.
To control the value in case the optional chainning is `undefined` you can combine the line with the nullish operator `??` and ensure that the value will be an empty string instead of undefined
```suggestion
        const text = event.clipboardData?.getData("text/plain") ?? "";
```